src/helpers.ts

```diff
@@ -6,22 +6,22 @@ import {
     SMS_SUBSCRIPTION_CHANGED
 } from './events';
 
-export function isValidCallback(handler) {
+export function isValidCallback(handler: any) { // type will be checked in this method
     invariant(
         typeof handler === 'function',
         'Must provide a valid callback'
     );
 }
 
-export function isObjectNonNull(object) {
+export function isObjectNonNull(object: any) {
```

Comment:
how about using a type guard?

https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards

```
function isObjectNonNull <T extends any>(object: T): obj is Exclude<T, null> {
    return object !== null;
}
```

Reply:
Should `undefined` also be excluded?